src/dev/config.js

```diff
@@ -427,11 +427,11 @@ module.exports = {
 
   nest: {
     // one or more thermostats
-    thermostats: [{ device: "structure/thermostat_name", name: "Hallway Thermostat" }],
+    thermostats: [{ device: "Falsetto/Hallway Thermostat", name: "Hallway Thermostat" }],
```

Comment:
Falsetto/Hallway Thermostat 

That's my home.

Maybe we can go through the dummy data and remove references to my home?

After talking to Andy Dennis and showing him my config.js, he said I am advertising all the electronics I have and my income level and so on.